jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/actions/CaseActionsPresenter.java

```diff
@@ -88,72 +89,109 @@ protected void loadCaseInstance(final CaseInstanceSummary cis) {
 
     protected void showDynamicAction(CaseActionType caseActionType) {
         switch (caseActionType) {
-           case DYNAMIC_USER_TASK: {
-                newActionView.show(caseActionType, () ->
-                        addDynamicUserTaskAction(
-                                newActionView.getTaskName(),
-                                newActionView.getDescription(),
-                                newActionView.getActors(),
-                                newActionView.getGroups(),
-                                newActionView.getStageId()));
+            case DYNAMIC_USER_TASK: {
+                newActionView.show(caseActionType,
+                                   () ->
+                                           addDynamicUserTaskAction(
```

Comment:
Just one small remark on this one.
I think that starting method call in new line when called like this doesn't enhance readability.